pom.xml

```diff
@@ -75,7 +75,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.4.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.5.0-SNAPSHOT</cdap.version>
```

Comment:
this doesn't match up with the lower bound specified in the parent. Can you change this to 6.1.1 (6.1.0 was never published)?